x/evm/types/key.go

```diff
@@ -39,6 +39,7 @@ const (
 var (
 	KeyPrefixCode    = []byte{prefixCode}
 	KeyPrefixStorage = []byte{prefixStorage}
+	KeyParams        = []byte{0x03}
```

Comment:
@Vvaradinov we should define a key for each of the fields of the params and store them individually in the KVStore